Dockerfile

```diff
@@ -7,7 +7,7 @@ EXPOSE 8080
 
 WORKDIR /app
 ADD Gemfile Gemfile.lock /app/
-RUN bundle install --deployment
+RUN bundle config set --local deployment 'true' && bundle install
```

Comment:
I don't think we need the quotes around `'true'` here. All CLI arguments get passed through as strings to Ruby.

Reply:
Yes, it could. This is what the bundler itself told me that is the preferred way :).